### PR TITLE
Bump navigation to get RTL fix

### PR DESCRIPTION
**Pull request by svanherk**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -25,17 +25,17 @@
       "d2l-icons": "5.1.5",
       "d2l-image-banner-overlay": "3.3.3",
       "d2l-image-selector": "3.1.0",
-      "d2l-inputs": "1.0.6",
+      "d2l-inputs": "1.0.7",
```
Comment (svanherk) [inline bower.json]: Fix for scroll always showing in IE11

File: bower.json [hunk 2]
```diff
@@ -12,7 +12,7 @@
       "d2l-course-image": "2.1.6",
       "d2l-dnd-sortable": "0.0.4",
       "d2l-dropdown": "6.6.5",
-      "d2l-enrollments": "0.3.24",
+      "d2l-enrollments": "0.3.25",
```
Comment (svanherk) [inline bower.json]: Updated langerms

File: bower.json [hunk 3]
```diff
@@ -25,17 +25,17 @@
...
       "d2l-image-banner-overlay": "3.3.3",
       "d2l-image-selector": "3.1.0",
-      "d2l-inputs": "1.0.6",
+      "d2l-inputs": "1.0.7",
       "d2l-intersection-observer-polyfill-import": "1.0.0",
       "d2l-intl-import": "1.0.0",
       "d2l-link": "4.0.2",
       "d2l-loading-spinner": "6.0.3",
       "d2l-localize-behavior": "1.1.4",
-      "d2l-menu": "1.2.1",
```
Comment (svanherk) [inline bower.json]: Change how returnItem is removed (FYI @margaree)

File: bower.json (hunk 2 above)
Comment (jeroach) [reply bower.json]: fyi @m6jones

**Pull request merged**